Purchases/Purchasing/StoreProduct.swift

```diff
@@ -106,10 +108,33 @@ public typealias SK2Product = StoreKit.Product
 
     @objc public override var localizedTitle: String { underlyingSK2Product.displayName }
 
+    /// Provides a `NumerFormatter`, useful for formatting the price for displaying.
```

Comment:
Nit: `NumberFormatter`
```suggestion
    /// Provides a `NumberFormatter`, useful for formatting the price for displaying.
```